pkg/term/term_windows.go

```diff
@@ -6,10 +6,10 @@ import (
 	"io"
 	"os"
 	"os/signal"
-	"syscall"
 
 	"github.com/Azure/go-ansiterm/winterm"
 	"github.com/docker/docker/pkg/term/windows"
+	"golang.org/x/sys/windows"
```

Comment:
windows failure is because this and the import above it both resolve to package windows. 

@jhowardmsft, because I want to make the change from syscall to -> golang.org/x/sys/(windows|unix etc) across the entire project, you have any objections to me changing the /pkg/term/windows package to something else (package windowsconsole or something)?